pages/home.html

```diff
@@ -100,10 +97,7 @@
     Contributing to Oppia
   </h2>
   <p>
-    Oppia is built by the community for the community. Our mission is to enable
-    anyone to learn anything they want to in an effective, enjoyable way, and
-    building a platform that allows anyone to create and share bite-sized
-    interactive learning material is just a first step.
+    Built by the community for the community, Oppia helps to learn anything you want to in an effective, enjoyable way. So far, we've developed a platform that allows people to create and share bite-sized interactive learning material, but this is just a first step. We're planning to add new features in order to bring you more interesting experiences with Oppia.
```

Comment:
The first sentence of this is technically incorrect -- we aren't actually living up to that promise yet.

Perhaps "Oppia aims to help you learn ..."?